app/kumactl/cmd/get/get_dataplanes.go

```diff
@@ -9,30 +9,31 @@ import (
 
 	"github.com/Kong/kuma/app/kumactl/pkg/output"
 	"github.com/Kong/kuma/app/kumactl/pkg/output/printers"
+	"github.com/Kong/kuma/app/kumactl/pkg/output/table"
 	"github.com/Kong/kuma/pkg/core/resources/apis/mesh"
 	rest_types "github.com/Kong/kuma/pkg/core/resources/model/rest"
 	core_store "github.com/Kong/kuma/pkg/core/resources/store"
 )
 
-func newGetDataplanesCmd(pctx *getContext) *cobra.Command {
+func newGetDataplanesCmd(pctx *listContext) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "dataplanes",
 		Short: "Show Dataplanes",
 		Long:  `Show Dataplanes.`,
-		RunE: func(cmd *cobra.Command, _ []string) error {
+		RunE: func(cmd *cobra.Command, x []string) error {
```

Comment:
Why that parameter became named?

Reply:
+1 I was testing something, forgot to rever it.